tests/src/Kernel/TranslationsTest.php

```diff
@@ -42,4 +40,5 @@ public function testNoTranslationsOnSetup() {
     $translations = $this->localStorage->findTranslation([]);
     $this->assertEqual(count($translations), 0, 'Found 0 translations strings in the database.');
```

Comment:
`assertEqual` is deprecated. 
Need to use `assertEquals`.
There are some other in this file too `drupal-potion/tests/src/Kernel/TranslationsImportTests.php`.